TotalSegmentator/TotalSegmentator.py

```diff
@@ -317,7 +317,10 @@ def __init__(self):
 
         ScriptedLoadableModuleLogic.__init__(self)
 
-        self.totalSegmentatorPythonPackageDownloadUrl = "https://github.com/wasserth/TotalSegmentator/archive/b38eb449ad8652a987878a925203cbfa354e9b85.zip"  # tag: 1.5.3
+        # self.totalSegmentatorPythonPackageDownloadUrl = "https://github.com/wasserth/TotalSegmentator/archive/b38eb449ad8652a987878a925203cbfa354e9b85.zip"  # tag: 1.5.3
+        #self.totalSegmentatorPythonPackageDownloadUrl = "https://github.com/wasserth/TotalSegmentator/archive/c0c4ecd8feb24d684aa919c20e35e8a1ed4ef617.zip"  # tag: 1.5.6
+        self.totalSegmentatorPythonPackageDownloadUrl = "https://github.com/wasserth/TotalSegmentator/archive/refs/tags/v1.5.7.zip"  # tag: 1.5.7
```

Comment:
We cannot trust git tags. They can be assigned to a different software version anytime and we have no way of knowing that our installed package is no longer up-to-date. Therefore, please use a git hash (and remove lines that are commented out).